scripts/path_plans.py

```diff
@@ -1,250 +1,140 @@
-import numpy as np
+from visualizations import plot_path_transforms
+from transformations import Transformations
 from shapes import Plane
-from math import radians
-from mpl_toolkits.mplot3d import Axes3D
-import matplotlib.pyplot as plt
-
-
-def _calc_unit_vec_between(vec_a, vec_b): # type: (list[float], list[float]) -> list[float]
-    """Finds the vector pointing from vec_a to vec_b and scales it to unit length"""
-    vec = np.subtract(vec_b, vec_a)
-    vec = vec / np.sqrt(np.sum(vec**2))
-    return vec.tolist()
+from shapes import Ring
+from typing import List
+from typing import Tuple
+from typing import Dict
```

Comment:
These imports are not used. Further they are very close to existing reserved names of list.
Why do these need included?

(same comment for similar imports in other files)

Reply:
[See above](https://github.com/osu-capstone-afrl/photogrammetry-capture-motion/pull/6#discussion_r602890426)